tutor/commands/compose.py

```diff
@@ -117,45 +130,99 @@ def convert(
         param: t.Optional["click.Parameter"],
         ctx: t.Optional[click.Context],
     ) -> t.List["MountType"]:
-        mounts: t.List["MountParam.MountType"] = []
+        mounts = self.convert_explicit_form(value) or self.convert_implicit_form(value)
+        if ctx:
+            mount_tmp_volumes(mounts, ctx.obj)
```

Comment:
> OK, this works, but it seems like we are leaning a little too hard on Click's internal behavior. Doing a stateful operation within convert() seems fishy to me.

Yes, I agree. It's annoying because I really pulled my hair trying to figure out what was the right place to put this `mount_tmp_volume` code, but you're right.

Thus we're back to the initial situation, where we need to manually call `mount_tmp_volumes` in every command. The problem that we are facing is that if we do that, then the `COMPOSE_CLI_MOUNTS` context needs to be different in dev and local. That's because `tutor dev start` triggers `tutor local stop`, and vice versa.

All of this is creating a lot of complexity, and I'd rather try to simplify things as much as possible. In order to avoid double calls, I suggest that the `quickstart` commands does not forward `--mount` arguments to its subcommands. I'll modify this PR to reflect this change, please let me know what you think.

Reply:
> In order to avoid double calls, I suggest that the quickstart commands does not forward --mount arguments to its subcommands. I'll modify this PR to reflect this change, please let me know what you think.

In retrospect, this is definitely the simplest solution. Sounds great.